distributions/distribution-resources/src/main/resources/conf/rules/readme.txt

```diff
@@ -0,0 +1,7 @@
+
+openHAB2 conf/rules folder
+
+Your rules go here.
+All rule files have to have the ".rules" file extension and must follow a special syntax.
+
+Check out the openHAB Wiki for more details: https://github.com/openhab/openhab/wiki/Rules
```

Comment:
For a change, I already ported this content to http://docs.openhab.org/features/automation/ruledsl.html, so you only need to update the url.